test/scenarios/cloudnative/network_problems/network_problems.go

```diff
@@ -31,11 +31,11 @@ const (
 )
 
 var (
-	csiNetworkPolicy = path.Join(project.TestDataDir(), "network/csi-denial.yaml")
+	csiNetworkPolicy = path.Join(project.TestDataDir(), "network_problems/csi-denial.yaml")
```

Comment:
some string replace gone wrong in this file for `network_problems` 😅 

(the file in this line does not exist, and some names and logs further down dont really need the `_`)

Reply:
nitpick: I still see a few comments that have the weird string replacement 😬 
- `builder := features.New("creating network_problems problems")`
- `namespaceBuilder := namespace.NewBuilder("network_problems-problem-sample")`
- `// Register network_problems policy to block csi driver traffic`
- `// Register network_problems-policy, sample, dynakube and operator uninstall`

Reply:
fixed in 4ecfac74